library/src/main/java/com/daimajia/numberprogressbar/NumberProgressBar.java

```diff
@@ -11,66 +11,64 @@
 import android.util.AttributeSet;
 import android.view.View;
 
+import static com.daimajia.numberprogressbar.NumberProgressBar.ProgressTextVisibility.Invisible;
+import static com.daimajia.numberprogressbar.NumberProgressBar.ProgressTextVisibility.Visible;
+
 /**
  * Created by daimajia on 14-4-30.
  */
 public class NumberProgressBar extends View {
 
-    private Context mContext;
-
-    /**
-     * The max progress, default is 100
-     */
-    private int mMax = 100;
+    private int maxProgress = 100;
 
     /**
-     * current progress, can not exceed the max progress.
+     * Current progress, can not exceed the max progress.
      */
-    private int mProgress = 0;
+    private int currentProgress = 0;
```

Comment:
I just found the convention not from Java coding style, but from [AOSP convention](http://source.android.com/source/code-style.html#follow-field-naming-conventions).
So it makes sense.